packages/react-migration-v8-v9/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@fluentui/react-migration-v8-v9",
-  "version": "1.0.0-alpha.0",
-  "private": true,
-  "description": "Migration shim components for hybrid v8/v9 apps.",
+  "version": "1.0.0",
+  "private": false,
```

Comment:
nit: `private` is `false` by default
```suggestion
```

Reply:
Also, unable to put suggestion since line wasn't changed but the beachball config should be changed to now allow `patch` and `minor` updates:
```
  "beachball": {
    "disallowedChangeTypes": [
      "major",
    ]
  },